modules/support-submitting-a-case-enable-approved-access.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * support/getting-support.adoc
+// * osd_architecture/osd-support.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="support-submitting-a-case-enable-approved-access_{context}"]
+= Submitting a support case to enable approved access for ROSA clusters
```

Comment:
Just a suggestion to put the most important part (enabling approved access) first

```suggestion
= Enabling approved access for ROSA clusters by submitting a support case
```